apidoc/Facebook.yml

```diff
@@ -1195,12 +1249,29 @@ properties:
     since: "5.4.0"
     platforms: [iphone, ipad]
 
+  - name: loginBehavior
+    summary: |
+        Defines the behavior that the module will attempt to use for authorize().
+    description: |
+        Be sure to set this before calling authorize if this is needed. It is
+        recommended to follow Facebook's [guidelines](https://developers.facebook.com/blog/post/2015/10/29/Facebook-Login-iOS9)
+        for ideal login behavior
+    type: Number
+    constants: [LOGIN_BEHAVIOR_*]
+    default:
+        Defaults to <Titanium.Modules.Facebook.LOGIN_BEHAVIOR_NATIVE_WITH_FALLBACK>
+        on Android and <Titanium.Modules.Facebook.LOGIN_BEHAVIOR_NATIVE> on iOS
```

Comment:
Not sure if I can follow. You are doing `[loginManager setLoginBehavior:FBSDKLoginBehaviorBrowser];` below which results in `Titanium.Modules.Facebook.LOGIN_BEHAVIOR_BROWSER`, so the default is browser not native. I need to do a feature test...